.github/workflows/wheels.yml

```diff
@@ -0,0 +1,63 @@
+name: Build wheels
+
+on: [push]
+# workflow_dispatch:{}
+
+env:
+  DEBIAN_FRONTEND: noninteractive
+  FORCE_CUDA: 1
+  # all values:
+  # 3.5;3.7;5.0;5.2;5.3;6.0;6.1;6.2;7.0;7.2;7.5;8.0;8.6
+  # with optional +PTX
+  # https://github.com/pytorch/vision/blob/0610b13ac4af3717f538454a9c6b1f441cb386f3/packaging/pkg_helpers.bash#L63
+  # you need at least cuda 5.0 for some of the stuff compiled here.
+  #TORCH_CUDA_ARCH_LIST: "6.0 6.1 7.0 7.5 8.6"
+  TORCH_CUDA_ARCH_LIST: "6.1"
+
+jobs:
+  build_wheels:
+    strategy:
+      matrix:
+        # python: ["3.7", "3.8", "3.9", "3.10"]
```

Comment:
Is pip currently not available for 3.7, 3.8, 3.9?

Reply:
Quite the opposite, it is possible to compile to all of the above versions, this was just a test